QueryCiv3/GameSections.cs

```diff
@@ -55,7 +55,7 @@ public WrldSection(SavData sav)
         }
         public short ContinentCount { get => Data.Sav.ReadInt16(Offset1+4); }
         public int WsizID { get => Data.Sav.ReadInt32(Offset1+234); }
-        public WsizSection WorldSize { get => Data.Bic.Wsiz[WsizID]; }
+        public Biq.WSIZ WorldSize { get => Data.Bic.Wsiz[WsizID]; }
         public int Height { get => Data.Sav.ReadInt32(Offset2+8); }
```

Comment:
Correct me if I'm wrong: Aren't most of these rendered obsolete by the new sections, once we refactor references to use the new code?

It's okay if that's coming in the future, just wanted to check whether the majority of this file remains necessary since my first thought is it probably isn't anymore.

Reply:
For right now, I haven't really looked at much of the SAV logic other than changing around some names to make sure the code still compiles, so I can't say for sure. But my plan is that once I'm done filling out the BIQ sections, I'll work on implementing the same struct-memory-copying strategy for SAV, so that'll involve the refactoring, yeah.

Reply:
Ah yes, I see now.  All the updates are for the BIQ contents, which in some cases (CITY, TILE, etc.) have equivalents in the SAV.  Makes sense now.